docs/docs/references/user-attributes.mdx

```diff
@@ -63,14 +63,17 @@ When referencing user attributes in SQL you can use the following [SQL variables
 
 You can use user attributes to filter the rows returned by a query. This is useful if you want to restrict the data 
 based on the user's attributes. To reference a user attribute in your sql, use the special lightdash reference 
-`${lightdash.attributes.<attribute_name> }`. For example, if you have a user attribute called `sales_region` you can
+`${lightdash.attributes.<attribute_name> }`.
+We recommend using the `IN` operator because the attribute might have multiple values.
```

Comment:
Instead of recommend, say that this is the way to do it. The docs will be easier to read if it's opinionated!